src-go/server/cmd/main.go

```diff
@@ -11,14 +11,32 @@ import (
 )
 
 func main() {
-	addr := flag.String("a", server.DefaultAddress, "Address to listen on ([ip:]port)")
+	interceptAddr := flag.String("intercept", server.DefaultInterceptProxyAddress, "Intercept proxy address to listen on ([ip:]port)")
+	burpAddr := flag.String("burp", server.DefaultBurpProxyAddress, "Burp proxy address to listen on ([ip:]port)")
+	emulateAddr := flag.String("emulate", server.DefaultEmulateProxyAddress, "Emulate proxy address to listen on ([ip:]port)")
```

Comment:
I don't really like the name `emulate` because it's technically not emulating anything. Perhaps `spoof` would be a better choice of words? Or stick to the old name and just call it `server` :shrug:

Reply:
Done